src/imitation/scripts/parallel.py

```diff
@@ -72,19 +72,22 @@ def parallel(
         init_kwargs: Arguments to pass to `ray.init`.
         local_dir: `local_dir` argument to `ray.tune.run()`.
         upload_dir: `upload_dir` argument to `ray.tune.run()`.
-        search_alg: can be either "optuna" or None.
+        search_alg: can be either "optuna" or None. Setting `None` allows for
```

Comment:
Now it is a lot clearer what this parameter does. It is still confusing why it behaves that way. Again, I agree that we should keep the `repeat` argument but I think there should be a way to go without the `search_alg` argument.

Reply:
See the comment above: https://github.com/HumanCompatibleAI/imitation/pull/675#discussion_r1266249803